plugins/org.jkiss.dbeaver.model/src/org/jkiss/dbeaver/model/impl/jdbc/JDBCUtils.java

```diff
@@ -270,6 +270,17 @@ public static boolean safeGetBoolean(ResultSet dbResult, int columnIndex)
         }
     }
 
+    public static boolean safeGetBoolean(ResultSet dbResult, int columnIndex, String trueValue)

```

Comment:
I understand that you have copy-pasted this method, but let's add some annotations to season it well.

Let's mark `dbResult` and `trueValue` as `@NotNull`, as well move `{` onto the same line where the method's name lays.